includes/db/LoadBalancer.php

```diff
@@ -341,13 +341,8 @@ function getReaderIndex( $group = false, $wiki = false ) {
 		}
 
 		if ( $i !== false ) {
+			# Wikia change - SUS-1801: don't abuse MASTER_POS_WAIT when obtaining slave connection
 			# Slave connection successful
-			# Wait for the session master pos for a short time
-			if ( $this->mWaitForPos && $i > 0 ) {
-				if ( !$this->doWait( $i ) ) {
-					$this->mServers[$i]['slave pos'] = $conn->getSlavePos();
-				}
-			}
 			if ( $this->mReadIndex <=0 && $this->mLoads[$i]>0 && $i !== false ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)